src/main/java/cc/polyfrost/oneconfig/internal/renderer/NanoVGHelperImpl.java

```diff
@@ -112,12 +112,13 @@ public static void setupAndDraw(boolean mcScaling, LongConsumer consumer) {
      * @param height The height.
      * @param color  The color.
      */
-    public static void drawRect(long vg, float x, float y, float width, float height, int color) {
+    @Override
+    public void drawRect(long vg, float x, float y, float width, float height, int color) {
```

Comment:
see we could remove the vg parameter now on all these methods in theory and just implement finalize/add a close method

I don't know how practical that is with the impl though just a suggestion.

Reply:
Yeah I didn't want to touch these kind of things yet, since that should be done at a later time/after merging, but I also think we should store the nvg context somewhere or something to have better syntax and less "boilerplate"

Reply:
okay, you could mark this as apistatus.internal for now if we plan Todo large changes (polyui)

Reply:
I think it's already done in #138